docs/user-guide/release-notes/master.rst

```diff
@@ -23,11 +23,12 @@ New Features
 Deprecation
 -----------
 
-Supported Platforms Changes
+Dependency/Platform Changes
 ---------------------------
 
 * If run on CentOS or Red Hat Enterprise Linux, the Pulp server now requires either
   version 7.1+ or 6.7+.
+* pymongo >= 3.0.0 is now required.
```

Comment:
Technically we don't require 3.2, just 3.0. We are shipping 3.2, but I don't think we need to mention that in our upstream release notes, which I think should target the Python audience more than the repo users.